.github/workflows/test-build-deploy.yml

```diff
@@ -80,10 +82,97 @@ jobs:
           mkdir /tmp/images
           ln -s /tmp/images ./docker-images
           make BUILD_IN_CONTAINER=false save-images
-      - name: Zip Images
-        run: tar -zcvf images.tar.gz /tmp/images
-      - name: Upload Images Artifact
+      - name: Create Docker Images Archive
+        run: tar -cvf images.tar.gz /tmp/images
```

Comment:
Given we're not compressing anymore, could you remove the `.gz` suffix?

Reply:
Removed